misc/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordDirectConverter.java

```diff
@@ -71,6 +71,9 @@ public class JsonToRecordDirectConverter {
 
     private JsonToRecordDirectConverter() {}
 
+    private static final String newRecordName = "NewRecord";
+    private static final String arrayRecordSuffix = "Item";
+
```

Comment:
Shall we remove an additional new line?

Reply:
This new line is to differentiate methods and properties right. Is that also need to be removed?